osu.Game.Rulesets.Mania/Skinning/Legacy/LegacyManiaJudgementPiece.cs

```diff
@@ -56,31 +56,30 @@ public void PlayAnimation()
 
             (animation as IFramedAnimation)?.GotoFrame(0);
 
+            this.FadeInFromZero(20, Easing.Out)
+                .Then().Delay(160)
+                .FadeOutFromOne(40, Easing.In);
+
             switch (result)
             {
                 case HitResult.None:
                     break;
 
                 case HitResult.Miss:
-                    animation.ScaleTo(1.6f);
-                    animation.ScaleTo(1, 100, Easing.In);
-
-                    animation.MoveTo(Vector2.Zero);
-                    animation.MoveToOffset(new Vector2(0, 100), 800, Easing.InQuint);
+                    animation.ScaleTo(1.2f).Then().ScaleTo(1, 100, Easing.Out);
 
                     animation.RotateTo(0);
-                    animation.RotateTo(40, 800, Easing.InQuint);
-
-                    this.FadeOutFromOne(800);
+                    animation.RotateTo(RNG.NextSingle(-18, 18), 100, Easing.Out);
```

Comment:
This seems like way too much rotation - it's almost a 40 degree random spread, stable definitely doesn't go that wide. Feels like this should be about a third of the current range at most.

Reply:
i converted wrong, should be 5.73 instead (stable range is -0.1 to 0.1 radians)